std_msgs/Header header
float32 slope
float32 intercept